Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml

```diff
@@ -0,0 +1,530 @@
+      playbookName: File Enrichment - VMRay
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      MD5:
+        complex:
+          root: File
+          accessor: MD5
+      SHA1:
+        complex:
+          root: File
+          accessor: SHA1
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 710
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "22":
+    id: "22"
+    taskid: a6c49ec1-43a1-42d5-8a86-cb443e867b13
+    type: playbook
+    task:
+      id: a6c49ec1-43a1-42d5-8a86-cb443e867b13
+      version: -1
+      name: Valid SHA256
+      description: Test file enrichment with a valid SHA256 hash
+      playbookName: File Enrichment - VMRay
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "23"
+    scriptarguments:
+      MD5:
+        complex:
+          root: File
+          accessor: MD5
+      SHA1:
+        complex:
+          root: File
+          accessor: SHA1
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 960,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "23":
+    id: "23"
+    taskid: f05cadf1-2f04-4f69-8402-ab2b91218f67
+    type: playbook
+    task:
+      id: f05cadf1-2f04-4f69-8402-ab2b91218f67
+      version: -1
+      name: Unknown SHA256
+      description: Test file enrichment with an unknown SHA256 hash
+      playbookName: File Enrichment - VMRay
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "24"
+    scriptarguments:
+      MD5:
+        complex:
+          root: File
+          accessor: MD5
+      SHA1:
+        complex:
+          root: File
+          accessor: SHA1
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 960,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "24":
+    id: "24"
+    taskid: 9b100dcd-026d-4014-8eaf-74afff2085b1
+    type: playbook
+    task:
+      id: 9b100dcd-026d-4014-8eaf-74afff2085b1
+      version: -1
+      name: Invalid SHA256
```

Comment:
Please add the condition that the playbook won't stop on errors, as we discussed

Reply:
Added the `continueonerror` property in f1489203a22035

Reply:
The test playbook succeeds now:
![image](https://user-images.githubusercontent.com/67368874/133070566-c47ecbad-5507-4e9b-80d3-f314c788da17.png)